plutarch-ledger-api/src/Plutarch/LedgerApi/V1.hs

```diff
@@ -174,11 +174,10 @@ newtype PTxInfo (s :: S)
                , "fee" ':= Value.PValue 'AssocMap.Sorted 'Value.Positive
                , "mint" ':= Value.PValue 'AssocMap.Sorted 'Value.NonZero -- value minted by transaction
                , "dCert" ':= PBuiltinList (PAsData DCert.PDCert)
-               , "wdrl" ':= AssocMap.PMap 'AssocMap.Unsorted Credential.PStakingCredential PInteger -- Staking withdrawals
+               , "wdrl" ':= PBuiltinList (PAsData (PBuiltinPair (PAsData Credential.PStakingCredential) (PAsData PInteger))) -- Staking withdrawals
                , "validRange" ':= Interval.PInterval Time.PPosixTime
                , "signatories" ':= PBuiltinList (PAsData Crypto.PPubKeyHash)
-               , "redeemers" ':= AssocMap.PMap 'AssocMap.Unsorted Contexts.PScriptPurpose Scripts.PRedeemer
-               , "data" ':= AssocMap.PMap 'AssocMap.Unsorted Scripts.PDatumHash Scripts.PDatum
+               , "data" ':= PBuiltinList (PAsData (PBuiltinPair (PAsData Scripts.PDatumHash) (PAsData Scripts.PDatum)))
```

Comment:
This isn't quite ideal. We should fix it properly later.

Reply:
I'm not sure what we can even fix. This is the representation we're forced to use.